yolox-burn/src/model/yolox.rs

```diff
@@ -0,0 +1,344 @@
+use burn::{
+    module::Module,
+    tensor::{backend::Backend, Device, Tensor},
+};
+
+use super::{
+    head::{Head, HeadConfig},
+    pafpn::{Pafpn, PafpnConfig},
+};
+
+#[cfg(feature = "pretrained")]
+use {
+    super::weights::{self, WeightsMeta},
+    burn::record::{FullPrecisionSettings, Recorder, RecorderError},
+    burn_import::pytorch::{LoadArgs, PyTorchFileRecorder},
+};
+
+// TODO: YOLOX
+// Only YOLOX-Nano uses depthwise=True (DwsConv)
```

Comment:
This TODO still valid?

Reply:
Man I swear I always forget a dead comment somewhere 🤣 

The comment is not valid, I just added the Nano version with the recent support (thanks to you) for enums in deserialization.